stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/gallery/AttachmentGalleryActivity.kt

```diff
@@ -81,11 +81,15 @@ public class AttachmentGalleryActivity : AppCompatActivity() {
     }
 
     private fun setupGallery(attachmentGalleryItems: List<AttachmentGalleryItem>) {
-        this.attachmentGalleryItems = attachmentGalleryItems
-        setupGalleryAdapter()
-        setupShareImageButton()
-        setupAttachmentActionsButton()
-        observePageChanges()
+        if (attachmentGalleryItems.isEmpty()) {
```

Comment:
Strange that it's possible to open gallery when a collection is empty

Reply:
@ogkuzmin Probably empty after process death

Reply:
Yep, it happens because we are not adding attachments to the bundle (The list could be so big)
Normally a user will open the gallery, interact with it and close it. Event if they leave the app and open again, if the system didn't kill the process, they will be able to see the full gallery again. In another case, close the gallery I think is the easy and fast fix for it